krabs/krabs/filtering/event_filter.hpp

```diff
@@ -86,10 +103,15 @@ namespace krabs {
     // Implementation
     // ------------------------------------------------------------------------
 
-    inline event_filter::event_filter(filter_predicate predicate)
-    : predicate_(predicate)
+    inline event_filter::event_filter(filter_predicate predicate): 
+		predicate_(predicate)
     {}
 
+	inline event_filter::event_filter(unsigned short event_id, filter_predicate predicate/*=nullptr*/):
```

Comment:
```suggestion
	inline event_filter::event_filter(unsigned short event_id, filter_predicate predicate/*=nullptr*/)
	: provider_filter_event_id_(event_id)
	, predicate_(predicate)
```

Reply:
Please maintain the existing style of files.

Reply:
Done